tests/src/test/java/alluxio/shell/command/CheckConsistencyCommandTest.java

```diff
@@ -21,6 +21,8 @@
 import org.junit.Assert;
 import org.junit.Test;
 
+import java.io.OutputStream;
+
 public class CheckConsistencyCommandTest extends AbstractAlluxioShellTest {
```

Comment:
I have test the the inconsistent directory, when i delete "/testRoot/testDir" in ufs, the testDir is inconsistent.